java/client/src/main/java/glide/api/commands/ConnectionManagementClusterCommands.java

```diff
@@ -47,6 +62,11 @@ public interface ConnectionManagementClusterCommands {
      * @param route Routing configuration for the command. Client will route the command to the nodes
      *     defined.
      * @return <code>String</code> with a copy of the argument <code>message</code>.
+     * @example
+     *     <pre>{@code
+     * String payload = clusterClient.ping("GLIDE", RANDOM_NODE).get();
```

Comment:
RequestRoutingConfiguration.java show a RANDOM enum value, but not RANDOM_NODE, should this be RANDOM?
```suggestion
     * String payload = clusterClient.ping("GLIDE", RANDOM).get();
```